.github/workflows/deb-builder.yml

```diff
@@ -51,3 +51,11 @@ jobs:
           name: ${{ env.artifacts }}
           path: debian/artifacts/*
           retention-days: 90
+
+      - name: Release
+        # FIXME: use tag when https://github.com/softprops/action-gh-release/issues/410 is closed
+        uses: softprops/action-gh-release@4634c16e79c963813287e889244c50009e7f0981
```

Comment:
Reading the issue, it's just a warning about deprecation. Do you think it's worth fixing the hash instead of just leaving the `@v1` and wait the upcoming release to fix it.

My concern is that we will probably forget to update it and it'll be fixed until the next time we have to look at it.